modules/standard/RecordParser.chpl

```diff
@@ -94,9 +192,15 @@ class RecordReader {
     } while(once);
   }
 
-  // An internal function that we use all our user visible code.
-  // When called with no arguments it does no checking abut block boundaries
-  // HDFS specific when offst and len are given. Checks for block boundaries for parallel file IO
+  /*
+     An internal function that we use all our user visible code.  When called
+     with no arguments it does no checking abut block boundaries.
+     
+     HDFS specific when offst and len are given. Checks for block boundaries
+     for parallel file IO
```

Comment:
What is meant by "HDFS specific"?  I see this function is used in that manner by the stream_num iterator, does that mean that stream_num is "HDFS specific"?